src/frameworks.mjs

```diff
@@ -33,12 +33,29 @@ export default [
 			return [files.find(({ fileName }) => fileName === 'App.jsx'), ...(files.filter(({ fileName }) => fileName !== 'App.jsx') || [])].filter((x) => x);
 		},
 	},
+  {
+		id: 'vue2',
+		title: 'Vue 2',
+		img: 'framework/vue.svg',
+		eslint: {
+			files: ['**/vue2/*.vue'],
+			extends: ['eslint:recommended', 'plugin:vue/recommended'],
+			rules: {
+				'vue/multi-word-component-names': 'off'
+			},
+		},
+		playgroundURL: 'https://sfc.vuejs.org',
+		documentationURL: 'https://vuejs.org/guide',
```

Comment:
This is the documentation for Vue 3 only

Reply:
My fault, I have always regarded options api as vue2